examples/miral-kiosk/kiosk_main.cpp

```diff
@@ -79,6 +79,7 @@ struct KioskAuthorizer : miral::ApplicationAuthorizer
 };
 
 std::atomic<bool> KioskAuthorizer::startup_only{false};
+
```

Comment:
Unrelated?

Reply:
removed accidental extra line in pr update